forgerock-auth/src/main/java/org/forgerock/android/auth/OAuth2Client.java

```diff
@@ -211,13 +211,14 @@ public void onResponse(@NotNull Call call, @NotNull Response response) {
      * @param accessToken The AccessToken to be revoked
      * @param listener    Listener to listen for revoke event
      */
-    public void revoke(@NonNull AccessToken accessToken, final FRListener<Void> listener) {
+    public void revoke(@NonNull AccessToken accessToken, boolean useRefreshToken, final FRListener<Void> listener) {
```

Comment:
Update javadoc to include new `useRefreshToken` parameter. Does the new parameter introduces a break change?

Reply:
Javadoc added